src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -651,37 +823,112 @@ subroutine driver_pbl(itimestep,configs,mesh,sfc_input,diag_physics,tend_physics
+                  qbuoy      = qbuoy_p      , qdiss    = qdiss_p    , sh3d        = sh3d_p           , &
+                  sm3d       = sm3d_p       , spp_pbl  = spp_pbl    , pattern_spp = pattern_spp_pbl  , &
+                  do_restart         = config_do_restart   ,                                           &
+                  do_DAcycling       = config_do_DAcycling ,                                           &
+                  initflag           = initflag            ,                                           &
+                  bl_mynn_tkeadvect  = bl_mynn_tkeadvect   ,                                           &
+                  bl_mynn_tkebudget  = bl_mynn_tkebudget   ,                                           &
+                  bl_mynn_cloudpdf   = bl_mynn_cloudpdf    ,                                           &
+                  bl_mynn_mixlength  = bl_mynn_mixlength   ,                                           &
+                  bl_mynn_closure    = bl_mynn_closure     ,                                           &
+                  bl_mynn_stfunc     = bl_mynn_stfunc      ,                                           &
+                  bl_mynn_topdown    = bl_mynn_topdown     ,                                           &
+                  bl_mynn_scaleaware = bl_mynn_scaleaware  ,                                           &
+                  bl_mynn_dheat_opt  = bl_mynn_dheat_opt   ,                                           &
+                  bl_mynn_edmf       = bl_mynn_edmf        ,                                           &
+                  bl_mynn_edmf_dd    = bl_mynn_edmf_dd     ,                                           &
+                  bl_mynn_edmf_mom   = bl_mynn_edmf_mom    ,                                           &
+                  bl_mynn_edmf_tke   = bl_mynn_edmf_tke    ,                                           &
+                  bl_mynn_output     = bl_mynn_edmf_output ,                                           &
+                  bl_mynn_mixscalars = bl_mynn_mixscalars  ,                                           &
+                  bl_mynn_cloudmix   = bl_mynn_cloudmix    ,                                           &
+                  bl_mynn_mixqt      = bl_mynn_mixqt       ,                                           &
+                  ids = ids , ide = ide , jds = jds , jde = jde , kds = kds , kde = kde ,              &
+                  ims = ims , ime = ime , jms = jms , jme = jme , kms = kms , kme = kme ,              &
+                  its = its , ite = ite , jts = jts , jte = jte , kts = kts , kte = kte ,              &
+                  errmsg = errmsg , errflg = errflg                                                    &
+                          )
+       call mpas_timer_stop('bl_mynn')
+       call mpas_log_write('--- exit subroutine mynn_bl_driver:')
+       call mpas_log_write(' ')
```

Comment:
For these and similar print statements, can these be removed?

Reply:
As requested, I commented out the extra print statements.